hack/release.sh

```diff
@@ -29,6 +29,8 @@ update_chart_yamls() {
 
     sed -i "s/version.*/version: ${version}/" helm/botkube/Chart.yaml
     sed -i "s/appVersion.*/appVersion: ${version}/" helm/botkube/Chart.yaml
+    sed -i "s/\btag:.*/tag: ${version}/" helm/botkube/values.yaml
+    sed -i "s/\bpullPolicy:.*/pullPolicy: IfNotPresent/" helm/botkube/values.yaml
```

Comment:
No need for this statement since we have already set `pullPolicy: IfNotPresent` and that is not going to change even after releases

Reply:
I thought that with `latest` it should be `Always`.
Not sure if the values.yaml file should contain `tag: latest` and `pullPolicy: Always` in `develop`.

Reply:
If image tag is set to latest, the image will always be pulled irrespective of what image policy is set. Ref: https://kubernetes.io/docs/concepts/containers/images/#updating-images. So we can just set pullPolicy to `IfNotPresent` in default helm values

Reply:
You're right, I forgot about the special handling of `latest` in k8s.
Done